.github/ISSUE_TEMPLATE/rebase.md

```diff
@@ -92,29 +98,46 @@ Branching is when a new stream is "branched" off of `rawhide`. This eventually b
 ```
 f32key=12c944d0
 key=$f32key
-untaglist=''
+> untaglist
```

Comment:
Minor: I think `touch untaglist` is more canonical. Or if you want to keep the semantic of "truncate if it exists", maybe just `echo > untaglist`? `xargs` ignores blank lines.

Reply:
done